<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LopetusDialog</class>
 <widget class="QDialog" name="LopetusDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="peliPaattyiLabel">
     <property name="font">
      <font>
       <family>Minion Pro Cond</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Peli päättyi.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="kerasitLabel">
     <property name="text">
      <string>Keräsit pelin aikana</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLCDNumber" name="pisteet">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>1</number>
       </property>
       <property name="digitCount">
        <number>7</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pistettaLabel">
       <property name="text">
        <string>pistettä!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>26</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="top10Label">
     <property name="text">
      <string>Ylsit tuloksellasi Top10 listan sijalle X, onneksi olkoon!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>30</number>
     </property>
     <item>
      <widget class="QLabel" name="nimimerkkiLabel">
       <property name="text">
        <string>Syötä nimimerkki:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nimimerkkiKentta"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>26</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="lopetaButton">
     <property name="text">
      <string>Tallenna ja lopeta</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
